flumes/schema.py

```diff
@@ -100,7 +99,7 @@ class Info(Base):
     id = Column(Integer, primary_key=True)
     file_id = Column(Integer, ForeignKey("files.id", ondelete="CASCADE"))
 
-    duration = Column(BigInteger)
+    duration = Column(Integer)
```

Comment:
I understand this change should not land in the PR right?

Reply:
I made this change in the last PR. I have rewritten it as it was and made the change in the migration.